cmd/earthly/main.go

```diff
@@ -563,6 +564,13 @@ func newEarthlyApp(ctx context.Context, console conslogging.ConsoleLogger) *eart
 			Usage:       "Disable collection of analytics",
 			Destination: &app.disableAnalytics,
 		},
+		&cli.StringFlag{
+			Name:        "feature-flag-overrides",
+			EnvVars:     []string{"EARTHLY_FEATURE_FLAG_OVERRIDES"},
+			Usage:       "Apply additional flags after the VERSION command is parsed, multiple flags can be seperated by commas",
+			Destination: &app.featureFlagOverrides,
+			Hidden:      true, // used for feature-flipping from ./earthly dev script
+		},
```

Comment:
Would these apply to all Earthfiles encountered during the build?

Reply:
yes, does something like `Override all VERSION commands with these additional flags, multiple flags can be seperated by commas` make it more clear?

Reply:
Yes - that makes sense